### PR TITLE
PropertiesTrait allow set Handlers properties by config properties key

### PR DESCRIPTION
**Pull request by rollo-smartframe**

### CODE REVIEW

File: src/HandlerFactory/PropertiesTrait.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+namespace SmartFrame\Logger\HandlerFactory;
+
+use Monolog\Handler\HandlerInterface;
+
+trait PropertiesTrait
+{
+    protected function applyProperties(HandlerInterface $handler, array $properties): void
+    {
+        foreach ($properties as $property => $value) {
+            $setter = sprintf('set%s', ucfirst($property));
```
Comment (michalkorotkiewicz) [inline src/HandlerFactory/PropertiesTrait.php:14]: what if method not exists?

Comment (michalkorotkiewicz) [reply src/HandlerFactory/PropertiesTrait.php:14]: Maybe own exception class?

**Review by tobo-pl — APPROVED** [review]

**Review by grzegorzplech — APPROVED** [review]

File: src/HandlerFactory/PropertiesTrait.php (hunk 1 above)
Comment (rollo-smartframe) [reply src/HandlerFactory/PropertiesTrait.php:14]: @michalkorotkiewicz if the method does not exist there is an Error exception with the message "Call to undefined method class::method" I think this is fine.

Comment (michalkorotkiewicz) [reply src/HandlerFactory/PropertiesTrait.php:14]: ok

**Review by michalkorotkiewicz — APPROVED** [review]

**Pull request merged**